compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/Scrollbar.desktop.kt

```diff
@@ -472,7 +472,11 @@ private class LazyScrollbarAdapter(
     }
 
     override fun maxScrollOffset(containerSize: Int) =
-        (averageItemSize * itemCount - containerSize).coerceAtLeast(0f)
+        (averageItemSize * itemCount
```

Comment:
1. Not sure, if it is related to this fix or not. I checked with different values in the `run1` configuration, and the scrollbar sometimes has a wrong offset:

a gap bellow the scrollbar:
![image](https://user-images.githubusercontent.com/5963351/211371594-a6bb4c90-be17-497c-8e2c-45d5382d0778.png)

the scrollbar is cut at the bottom:
![image](https://user-images.githubusercontent.com/5963351/211371776-e5017d39-7e79-40af-8dbb-784205b54a15.png)

2. When I use `item`'s instead of `contentPadding`, found another bug - scrollbar stops moving, not reaching the bottom or the end:
```
    val itemCount = 46
    val heights = remember {
        val random = Random(24)
        (0 until itemCount).map { random.nextFloat() }
    }

    LazyColumn(modifier.graphicsLayer(alpha = 0.5f), state = state) {
        item {
            Box(Modifier.height(300.dp))
        }
        items((0 until itemCount).toList()) { i ->
            val itemHeight = 20.dp + 20.dp * heights[i]
            Text(i.toString(), Modifier.graphicsLayer(alpha = 0.5f).height(itemHeight))
        }
        item {
            Box(Modifier.height(10.dp))
        }
    }
```

https://user-images.githubusercontent.com/5963351/211373309-83b77e82-99f7-4749-9f32-143bb66a61fb.mp4

Probably, we should calculate `positionDuringDrag` in a more complex way.

Reply:
If the first bug is unrelated to the PR (the second bug is unrelated for sure), we can merge this now.

Reply:
The change is correct from my perspective, but maybe we miss something.

Reply:
2. Found, that you already described it in https://github.com/JetBrains/compose-jb/issues/1211#issuecomment-1374494944

Reply:
@igordmn Yes, (1) is probably the 3rd bug I talked about, that I think is related to `LazyScrollbarAdapter.scrollOffset` being calculated via `scrollState.firstVisibleItemIndex` and `averageItemSize` being calculated via `scrollState.layoutInfo.visibleItemsInfo`, which don't always match. I'll verify it.